### ISSUE TITLE
No element meets colour contrast

### ISSUE THREAD

**Issue by selfthinker** [ctomczyk/aslint#10]
Every element on every page on GDS' [Accessibility Tool Audit](https://alphagov.github.io/accessibility-tool-audit/) I test with ASLint apparently doesn't meet the colour contrast. Take a [very simple page](https://alphagov.github.io/accessibility-tool-audit/tests/content-content-identified-by-location.html) for example with only a heading, a paragraph and two links. And ASLint marks every single of those elements as not meeting the colour contrast requirements:

> Specified element has the background as semi-transparent, so the contrast ratio cannot be precise. Depending on what's going to be underneath, it could be any of the following: 1) fails WCAG 2.0, 2) passes AA for large text (above 18pt or bold above 14pt), 3) Passes AA level for any size text and AAA for large text (above 18pt or bold above 14pt), 4) Passes AAA level for any size text

It obviously shouldn't fail because you cannot get more contrast than with black on white.
I have no idea what is meant by "semi-transparent background". I've run the tool on some other pages and that didn't appear, so I wonder if the reason for this weird behaviour is the fact that no colours are set?

**Comment by ctomczyk** [comment]
This is happening because tested element in fact contains transparent background. For example, reported element `h1` contains transparent background and therefore there is a report like you mentioned. See the screenshot: https://www.aslint.org/temp/aslint_background.png

I was thinking about that testing before, but in reality what we see is different from what is rendered. That's why I took the elements rendered colours as they are rendered by the browser.

I am open to any other suggestions, anyway.

**Comment by ctomczyk** [comment]
This issue has been fixed with version 1.0.1029. Basically, if no background can be determined (means we have only transparent) then calculating contrast ratio for a particular HTML element is skipped.
